documentation/asciidoc/accessories/camera/libcamera_options_common.adoc

```diff
@@ -19,6 +19,23 @@ libcamera-apps build: ca559f46a97a 27-09-2021 (14:10:24)
 libcamera build: v0.0.0+3058-c29143f7
 ----
 
+----
+	--config,	-c		Read options from the given file <filename>
+----
+
+Normally options are read from the command line, but in case multiple options are required it may be more convenient to keep them in a file.
+
+Example: `libcamera-hello -c config.txt`
+
+This is a text file containing individual lines of `key=value` pairs, for example:
+
+----
+timeout=99000
+verbose=
+----
+
+Note how the `=` is required even for implicit options, and that the `--` used on the command line are omitted.
```

Comment:
Does it support both long- and short-options, or is it long-options only?

Reply:
Long only. Will update that!